apis/twitter.js

```diff
@@ -0,0 +1,42 @@
+import Twit from 'twit';
+import getObjProperty from 'lodash.get';
```

Comment:
I just meant to use a single module from lodash library instead of the entire package. What do you suggest ? :)